src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -83,7 +85,53 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func logPanic(proxyAddr string, msg string) {
+	cfg, err := config.Init(packageVersion)
+	if err != nil {
+		panic("Error initializing config")
+	}
+
+	// Create the client-side proxy.
+	c := &client.Client{
+		Addr:         proxyAddr,
+		ReadTimeout:  0, // don't timeout
+		WriteTimeout: 0,
+	}
+	ready := make(chan bool)
+	go c.ListenAndServe(func() { ready <- true })
+	<-ready
+
+	if err := logging.Init(); err != nil {
+		panic("Error initializing logging")
+	}
+
+	ready = logging.Configure(proxyAddr, cfg.CloudConfigCA, cfg.InstanceId, version, revisionDate)
+	<-ready
+
+	log.Error(msg)
+
+	logging.Flush()
+	_ = logging.Close()
+}
+
 func main() {
+	// panicwrap works by re-executing the running program (retaining arguments,
+	// environmental variables, etc.) and monitoring the stderr of the program.
+	exitStatus, err := panicwrap.BasicWrap(
+		func(output string) {
+			logPanic("127.0.0.1:18787", output)
+			exit(nil)
+		})
+	if err != nil {
+		// Something went wrong setting up the panic wrapper. This wont' be
+		// captured by panicwrap
+		log.Fatalf("Error setting up panic wrapper: %v", err)
```

Comment:
I think this should just be an Errorf. If we can't use panicwrap, we definitely want to know that in Loggly, but we also are no worse off than we were before so probably should not exit the program (which itself could well happen before we're able to log the error to Loggly).